src/components/SlateEditor/utils/defaultNormalizer.ts

```diff
@@ -28,201 +30,196 @@ export interface NormalizerConfig {
   nodes?: DefaultNodeRule;
 }
 
-const normalizeNodes = (editor: Editor, entry: NodeEntry, config: NormalizerConfig): boolean => {
-  const [node, path] = entry;
-
-  if (!Element.isElement(node)) return false;
-
-  const { firstNode, lastNode, nodes } = config;
-
-  const children = node.children;
+const normalizeNodes = (
+  editor: Editor,
+  element: Element,
+  path: Path,
+  config: NormalizerConfig,
```

Comment:
Har gjort et konsekvent valg av å alltid kalle `config` for config. Dropper å destrukturere objektet for å gjøre det eksplisitt hvor verdien kommer fra.